.circleci/config.yml

```diff
@@ -24,6 +23,11 @@ jobs:
       - run:
           name: Test images
           command: tests/run.sh
+      # Sleeping to ensure the supervisord has started.
+      - run: sleep 30s
+      - run:
+          name: Unit tests for clamav
+          command: docker-compose exec test sh -c 'cd /app/clamav && ./vendor/bin/phpunit'
```

Comment:
Is it possible to run this command in the goss test?

Reply:
Not really - goss is a serverspec tool to validating the configuration of a container. This is a unit test to make sure clam is responding how we expect.

Might be a little superfluous if goss validates the daemons are running and the port is open we can assume that clam will work as its intended to.

The unit test does give us a little assurance because it's in PHP it basically replicates exactly how the Drupal module sends files to clam over tcp.